src/lib/apiTypesV2/Subscription.h

```diff
@@ -75,8 +75,8 @@ struct Notification
   long long                lastNotification;
   HttpInfo                 httpInfo;
   std::string              toJson(const std::string& attrsFormat);
-  int                      lastFailure;
-  int                      lastSuccess;
+  int                      lastFailure;  // FIXME P4: should be long long
```

Comment:
I'd suggest add the ending "... as lastNotification is"

Reply:
Fixed in f114c3e